### PR TITLE
Improve update from 6.5 to 6.6

### PR DESCRIPTION
**Pull request by pweyck**
Change `bin/console`, `bin/ci` and `public/index.php` to be compatible with 6.6. This should improve the update flow a bit

### CODE REVIEW

**Comment by pweyck** [comment]
Still need to test this

**Comment by shyim** [comment]
I think you need to copy the `shopware/administration` `shopware/storefront` too for 6.5 version

File: shopware/core/6.5/public/index.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    $trustedHosts = $context['TRUSTED_HOSTS'] ?? false;
+    if ($trustedHosts) {
+        Request::setTrustedHosts(explode(',', $trustedHosts));
+    }
+
+    if (!EnvironmentHelper::getVariable('SHOPWARE_SKIP_WEBINSTALLER', false) && !file_exists(dirname(__DIR__) . '/install.lock')) {
+        return new InstallerKernel($appEnv, $debug);
+    }
+
+    $shopwareHttpKernel = new HttpKernel($appEnv, $debug, $classLoader);
```
Comment (shyim) [inline shopware/core/6.5/public/index.php:53]: you used the KernelFactory in the cli with an if condition, but not there. is that expected? 🤔

**Comment by shyim** [comment]
When the change is only the if condition with KernelFactory, we can do that also in the 6.4 variant. it's less duplicate then 😅

File: shopware/core/6.5/public/index.php (hunk 1 above)
Comment (pweyck) [reply shopware/core/6.5/public/index.php:53]: I'll change it in the index.php too

**Review by shyim — APPROVED** [review]

**Pull request merged**